nextflow.config

```diff
@@ -85,6 +88,7 @@ params {
 
     scpred{
         run = "False"
+        matrix_type = "" // must be either "raw", "filtered" or "normalised"
```

Comment:
Why empty?

Reply:
@pinin4fjords  I wasn't using this in testing, so forgot to fill in. We want CPM counts there, right?

Reply:
You said the authors' info said it didn't matter, so I'm not sure. But probably, yes